website/content/gateway/api_reference/changes/changes.md

```diff
@@ -10,6 +10,11 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### Add recipient to debits
+The `recipient` concept have been added to debits. The
+`recipient[account_number]` and `recipient[name]` is now required for merchant
+accounts with BAI AA from 2023-07-31.
```

Comment:
```suggestion
`recipient[account_number]` and `recipient[name]` parameters are now required
for merchant accounts with BAI AA from 2023-07-31.
```

Reply:
Alternative if we wish to "spell out" BAI and AA like it has been done for BAI and AA in the new `recipient_information.md`.

```suggestion
`recipient[account_number]` and `recipient[name]` parameters are now required
for merchant accounts with Business Application Identifier (BAI)
Account-to-Account (AA) from 2023-07-31.
```